extensions/stackdriver/common/context.h

```diff
@@ -0,0 +1,96 @@
+/* Copyright 2019 Istio Authors. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#pragma once
+
+#include <string>
+#include <unordered_map>
+
+#include "google/protobuf/struct.pb.h"
+
+namespace Extensions {
+namespace Stackdriver {
+namespace Common {
+
+// NodeInfo represents the information extracted from proxy node metadata, or
+// peer node metadata header. This is used to fill metrics and log labels.
+struct NodeInfo {
+  // Name of the node. e.g. pod name in k8s.
+  std::string name;
+
+  // Namespaace that the node runs in.
+  std::string namespace_name;
```

Comment:
namespace.

Reply:
This has been renamed to namespace in NodeInfo Protobuf.